test/assert.js

```diff
@@ -0,0 +1,42 @@
+(function() {
+    test_group = (name, test_group_function) => {
```

Comment:
bad name - use camel case for names! testGroupFunction

Reply:
Excersice's exmaple chose that name ..